src/upgrades/mod.rs

```diff
@@ -160,7 +158,17 @@ pub trait UpgradeStore {
 
 /// Should be called when Krill starts, before the KrillServer is initiated
 pub fn pre_start_upgrade(config: Arc<Config>) -> Result<(), UpgradeError> {
-    upgrade_0_9_0(config)
+    upgrade_data_to_0_9_0(config)
+}
+
+/// Should be called when Krill starts, before the KrillServer is initiated
+pub async fn post_start_upgrade(config: &Config, server: &KrillServer) -> Result<(), UpgradeError> {
+    if needs_upgrade(&config.data_dir, "cas", KrillVersion::candidate(0, 9, 3, 2)) {
+        info!("Reissue ROAs on upgrade to force short EE certificate subjects in the objects");
```

Comment:
"info", or should that be "warn" ?

Reply:
info, this is by design and no need to worry.